Server/controllers/pageSpeedCheckController.js

```diff
@@ -39,10 +39,26 @@ const getPageSpeedChecks = async (req, res, next) => {
  */
 const createPageSpeedCheck = async (req, res, next) => {
   try {
-    // Validate monitorId parameter
-    await createPageSpeedCheckParamValidation.validateAsync(req.params);
+    // Validate monitorId parameter and request body
+    const { monitorId } = req.params;
+    const { accessibility, bestPractices, seo, performance } = req.body;
+    
+    await createPageSpeedCheckParamValidation.validateAsync({ monitorId, accessibility, bestPractices, seo, performance });
```

Comment:
Only monitorId is a parameter of the request, the rest are part of the body and should have a separate validation schema